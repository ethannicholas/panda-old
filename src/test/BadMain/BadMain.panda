class BadMain {
    method main() {
    }

    @class
    method main(arg:String) {
    }

    @class
    method main(arg:PrimitiveArray<String>):Int {
    }
}