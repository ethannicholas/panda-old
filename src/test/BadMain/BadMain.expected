error: BadMain.panda:2:5: method 'main' must be declared '@class'
error: BadMain.panda:6:5: method 'main' must accept either no parameters or one parameter of type 'panda.collections.PrimitiveArray<panda.core.String>'
error: BadMain.panda:10:5: method 'main' may not return a value
3 errors
